client/question/services/code_evaluators/PythonCodeRunnerServiceSpec.js

```diff
@@ -64,6 +64,32 @@ describe('PythonCodeRunnerService', function() {
       $httpBackend.flush();
     });
 
+    it('processes a compile-time syntax error', function() {
+      var code = [
+        'def myFunction(arg):',
+        '    result = arg.rstrip()',
+        '    return result',
+        ''
+      ].join('\n');
+      responseDict.stderr = [
```

Comment:
Not clear how this stderr relates to the code above?

Reply:
They're unrelated -- this is just to mock out the functionality. The post returns responseDict on line 84 -- the code is just ... boilerplate, essentially.